PostalAddressRow.podspec

```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
     'PostalAddressRow' => ['Sources/**/*.xib']
   }
   # s.ios.frameworks = 'UIKit', 'Foundation'
-  s.dependency 'Eureka', '~> 4.0'
+  s.dependency 'Eureka'
```

Comment:
Could you please leave this as it was? If Eureka moves on to version 5 then PostalAddressRow might not be compatible with that.

Reply:
sure @mats-claassen I am on it. Will submit the updated Pull Request